services/usage.rst

```diff
@@ -310,9 +310,10 @@ GitHub
 PFS project uses `Subaru-PFS organization at GitHub <https://github.com/Subaru-PFS>`_. 
 In 'Subaru-PFS' organization at GitHub, 
 there is one (mostly private) `team per institute <https://github.com/orgs/Subaru-PFS/teams>`_
```

Comment:
> there  is one (mostly private) team per institute

LAM and PU have two teams for ICS and DRP separately.
how about changing as follows?

> there is basically one (mostly private) team per institute and development package, where...

Reply:
added.